composer.json

```diff
@@ -45,7 +45,9 @@
 		"phpunit/phpunit" : "^5.7",
 		"squizlabs/php_codesniffer": "^3.4",
 		"symfony/phpunit-bridge": "^4.4",
-		"symfony/web-profiler-bundle": "^4.4"
+		"symfony/web-profiler-bundle": "^4.4",
+		"phpstan/extension-installer": "^1.0",
```

Comment:
Mmmm what is this? Do we really need this, if so explain it to me :)

Reply:
I agree, the instructions also have a "manual" installation option, which is just adding 1 line of code to the phpstan configuration. The manual option is preferred